projects/igniteui-angular/migrations/update-15_0_4/index.spec.ts

```diff
@@ -24,18 +24,43 @@ describe(`Update to ${version}`, () => {
         },
         version: 1
     };
+    const configJsonLib = {
+        defaultProject: 'testProj',
+        projects: {
+            testProj: {
+                root: '/',
+                sourceRoot: '/testSrc',
+                prefix: 'lib',
```

Comment:
The prefix of a project of `projectType: 'library'` could be anything. The prefix of igniteui-angular is `igx`. The project type you're testing with here is the default one, which is `application`.

Reply:
Yes, in the schematics we had an enumeration that is checking against projectType, not the prefix, so I'm testing aginst it.